.rubocop.yml

```diff
@@ -200,7 +200,7 @@ Rails/ReadWriteAttribute:
 # post_scraper.rb is run manually in the command line so stdout is fine
```

Comment:
also this

Reply:
PostScraper is in fact the part that's run so it seemed more correct to leave that?

Reply:
Valid, could be ~any more clear that post_scraper calls reply_scraper but it's probably fine as is